analysis_driver/dataset.py

```diff
@@ -468,22 +466,24 @@ def non_useable_run_elements(self):
 
     @property
     def project_id(self):
-        return self.run_elements[0]['project_id']
+        return self.sample['project_id']
 
     def _amount_data(self):
-        return sum(
-            [
-                int(r.get(ELEMENT_NB_Q30_R1_CLEANED, 0)) + int(r.get(ELEMENT_NB_Q30_R2_CLEANED, 0))
-                for r in self.run_elements
-            ]
-        )
+        return int(self._sample['aggregated'].get('clean_yield_q30_in_gb')*1000000000)
+
+    @property
+    def pc_q30(self):
+        s = self.sample
+        reads_q30 = s.get(ELEMENT_NB_Q30_R1) + s.get(ELEMENT_NB_Q30_R2)
+        number_of_reads = s.get(ELEMENT_NB_BASE_R1) + s.get(ELEMENT_NB_BASE_R2)
+        return (reads_q30/number_of_reads) * 100
```

Comment:
These lines should go but FYI these values are number of bases not number of reads